api/client/amp.go

```diff
@@ -53,6 +53,8 @@ func (l logger) Println(args ...interface{}) {
 
 // Configuration is for all configurable client settings
 type Configuration struct {
+	Version            string
```

Comment:
Should Version and Build be part of the configuration? They will be displayed with the ```amp config``` command, and will be settable (is it a word?) with ```amp config Version value```.